Makefile

```diff
@@ -17,7 +17,7 @@ $(eval PI = $(shell cat /proc/cpuinfo 2>&1 | grep "Raspberry Pi"))
 all: clippy build test
 
 .PHONY: cross
-cross: cross-clippy cross-build cross-test cross-release-build
+cross: cross-clippy cross-build cross-test cross-release-build cross-build-pi3b cross-release-build-pi3b
```

Comment:
I think I would call the target arm7 instead of 3b, as that is the real change, and will probably work for other boards than a 3b also...